CHANGES.md

```diff
@@ -9,6 +9,7 @@ Breaking changes:
 
 - #3246: Badge color not responsive to dark theme
 - Fix a GIF rendering bug that causes a memory overflow
+- #2716: Fix issue with chat display when opening via URL
```

Comment:
Please move this line to under the latest release heading `## 11.0.0 (Unreleased)`

Other than that, looks good thanks!

Reply:
@jcbrand  I've moved the line, please let me know if there are any other changes needed.

Reply:
Thanks, however this isn't a breaking change, so your line needs to go above the `Breaking changes` heading.

Reply:
@jcbrand  moved it, is it correct now ?